docs/content/manual/manual.yml

```diff
@@ -79,8 +79,8 @@ sections:
       jq filters run on a stream of JSON data. The input to jq is
       parsed as a sequence of whitespace-separated JSON values which
       are passed through the provided filter one at a time. The
-      output(s) of the filter are written to standard out, again as a
-      sequence of whitespace-separated JSON data.
+      output(s) of the filter are written to standard output, as a
+      sequence of newline-separated JSON data.
```

Comment:
We don't have to mention carriage return on Windows, do we?  (No, we probably shouldn't.)